tests/rust/tcp-test/close/wait.rs

```diff
@@ -305,14 +309,16 @@ fn wait_for_accept_after_issuing_async_close(libos: &mut LibOS, local: &SocketAd
     }
 
     // Wait again on accept() and ensure that ECANCELED is returned this time.
-    match libos.wait(qt, Some(Duration::from_micros(0))) {
-        Ok(qr) if qr.qr_opcode == demi_opcode_t::DEMI_OPC_FAILED && qr.qr_ret == libc::ECANCELED as i64 => {},
-        // If we found a connection to accept, something has gone wrong.
-        Ok(qr) if qr.qr_opcode == demi_opcode_t::DEMI_OPC_ACCEPT && qr.qr_ret == 0 => {
-            anyhow::bail!("accept() should not succeed because remote should not be connecting")
-        },
-        Ok(_) => anyhow::bail!("wait() should return an error on accept() after close()"),
-        Err(e) => anyhow::bail!("wait() should not time out. {:?}", e),
+    if !accepted_completed {
+        match libos.wait(qt, Some(Duration::from_micros(0))) {
+            Ok(qr) if qr.qr_opcode == demi_opcode_t::DEMI_OPC_FAILED && qr.qr_ret == libc::ECANCELED as i64 => {},
+            // If we found a connection to accept, something has gone wrong.
+            Ok(qr) if qr.qr_opcode == demi_opcode_t::DEMI_OPC_ACCEPT && qr.qr_ret == 0 => {
```

Comment:
Can you remove the curly braces here too?

Reply:
Sorry, this is not possible because that makes the line longer than 120 characters long, which violates current rust formatting rules. Actually, even if I remove, rust formatter places it back.